Scellecs.Morpeh/Core/Worlds/WorldStashExtensions.cs

```diff
@@ -49,15 +49,16 @@ public static Stash<T> GetStash<T>(this World world) where T : struct, IComponen
             world.ThreadSafetyCheck();
             
             var info = ComponentId<T>.info;
-            
+            var capacity = ComponentId<T>.StashSize;
```

Comment:
Should be below the check, not above, as it's used in the constructor down the method